crates/next-core/js/src/entry/router.ts

```diff
@@ -65,7 +68,16 @@ async function getResolveRoute(
     true
   );
 
-  return await makeResolver(dir, nextConfig);
+  const edgeInfo = {
+    name: "edge",
+    paths: middlewareChunkGroup.map((chunk: string) =>
```

Comment:
Shouldn't this be called `files` and be relative to the `distDir` to match our middleware plugin 
x-ref: https://github.com/vercel/next.js/blob/7d8f85bd8dbd767dfac0d92eed651877268e62cb/packages/next/src/build/webpack/plugins/middleware-plugin.ts#L35

Reply:
I copied this from https://github.com/vercel/turbo/blob/805a5ae14f1ab67f3c3e01e758f7b53d44f9d696/crates/next-core/js/src/entry/server-edge-api.tsx#L53-L59, but I'm not sure if it's correct. @sokra?

Reply:
I aligned with the return value of `getEdgeFunctionInfo`, which converts the format in the manifest file into one with `paths`

https://github.com/vercel/next.js/blob/7d8f85bd8dbd767dfac0d92eed651877268e62cb/packages/next/src/server/next-server.ts#L1640-L1654

So when this info object is provided it could skip calling `getEdgeFunctionInfo`.

Reply:
I see, yeah aligning on that return type seems alright